src/api/routes/channels/#channel_id/webhooks.ts

```diff
@@ -23,7 +23,7 @@ router.post("/", route({ body: "WebhookCreateSchema", permission: "MANAGE_WEBHOO
 
 	let { avatar, name } = req.body as { name: string; avatar?: string };
 	name = trimSpecial(name);
-	if (name === "clyde") throw new HTTPError("Invalid name", 400);
+	if (name.toLowerCase() === "clyde") throw new HTTPError("Invalid name", 400);
```

Comment:
Maybe for a different PR, but an instance owner definable name blacklist, possibly regex compatible would be pretty handy

Reply:
#892